internal/cli/test.go

```diff
@@ -183,7 +183,8 @@ auth0 test login <client-id> --connection <connection>`,
 			}
 
 			if isFirstRun {
-				cli.renderer.Infof("%s Login flow is working! Next, try downloading and running a Quickstart: 'auth0 quickstarts download %s'",
+				cli.renderer.Infof("Login flow is working!")
+				cli.renderer.Infof("%s Consider downloading and running a quickstart by running `auth0 quickstarts download %s`",
```

Comment:
```suggestion
				cli.renderer.Infof("%s Consider downloading and running a quickstart next, by running `auth0 quickstarts download %s`",
```

Wdyt? Personally I find this a bit nicer, the user logged in the first time, let's suggest what to do next.